R/z_pages.R

```diff
@@ -47,6 +47,7 @@ animint2pages <- function(plot.list, github_repo, commit_message = "Commit from
       stop(sprintf("Please run `install.packages('%s')` before using this function", pkg))
     }
   }
+
```

Comment:
please undo this empty line addition, and avoid similar changes in the future

Reply:
done